monitoring/tangle_logrotate.conf

```diff
@@ -0,0 +1,61 @@
+# This is config file for logrotate (https://linux.die.net/man/8/logrotate)
+# This will ensure our logs do not grow unbounded,
+# How to read this log file
+#
+# <logfile_name> {
+#   su <user> <user>
+#   rotate <how many previous versions to keep>
+#   size <MAX_SIZE_OF_LOG_FILE> 
+#   postrotate <action_taken_after_rotate>
+# }
+#
+# Make sure to include this file in /etc/logrotate.conf
+/home/ubuntu/collator_logs/standalone_err.log {
+       # Command is run as root
+       su root root 
+       # Keeps last two versions of log file
+       rotate 2 
+       # Creates a new log file when current log file reaches 50MB
+       size 50M
+       # restart the service after switching logfiles
+       postrotate
+           systemctl restart standalone.service 
+       endscript
+}
+
+/home/ubuntu/collator_logs/network_logs.log {
+       su root root
+       rotate 2
+       size 10M
+       postrotate
+           systemctl restart standalone.service
+       endscript
+}
+
+/home/ubuntu/collator_logs/network_logs.log.signing.log {
+       su root root
+       rotate 2
+       size 10M
+       postrotate
+           systemctl restart standalone.service
+       endscript
+}
+
+/home/ubuntu/collator_logs/network_logs.log.voting.log {
+       su root root
+       rotate 2
+       size 10M
+       postrotate
+           systemctl restart standalone.service
+       endscript
+}
+
+/home/ubuntu/collator_logs/network_logs.log.keygen.log {
+       su root root
+       rotate 2
+       size 10M
+       postrotate
+           systemctl restart standalone.service
```

Comment:
Restarting the service does make it down for quite sometime, no? is that a good idea? can we do something better without  a restart? Like docker does have that feature for example, without a restart, like a hot-swap or something.

Reply:
If the file is modified, the logs wont be appended anymore, I couldnt find another solution for this.

Reply:
can we merge this @shekohex